src/classes/ERR_Handler.cls

```diff
@@ -310,6 +310,28 @@ public class ERR_Handler {
         }
     }
 
+    /*******************************************************************************************************
+    * @description Stores errors in the database (if enabled), do not generate apex page messages and sends error notifications (if enabled).
+    * @param errorRecords The errors to store.
+    * @param context The context the errors were raised on.
+    * @return void
+    */
+    public static void sendErrorNotification(Exception e, String context) {
+
+        if (e == null) {
+            return;
+        }
+
+        Error__c error = createError(e, context);
+        if (UTIL_CustomSettingsFacade.getErrorSettings().Store_Errors_On__c) {
+            UTIL_Debug.debug('**** Storing errorRecords: ' + error);
```

Comment:
Is the debug statement needed for something specific? If not, it can be removed.